cmd/logs.go

```diff
@@ -3,11 +3,14 @@ package cmd
 import (
 		"fmt"
 		"strings"
+	  "time"
```

Comment:
Ya, I'm wondering if we should automatically run `go fmt` as part of the develop build?